src/qt/bitcoingui.cpp

```diff
@@ -536,12 +536,6 @@ void BitcoinGUI::createActions()
         });
     }
 #endif // ENABLE_WALLET
-
-    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_I), this), &QShortcut::activated, this, &BitcoinGUI::showInfo);
-    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_C), this), &QShortcut::activated, this, &BitcoinGUI::showConsole);
-    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_G), this), &QShortcut::activated, this, &BitcoinGUI::showGraph);
-    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_P), this), &QShortcut::activated, this, &BitcoinGUI::showPeers);
-    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_R), this), &QShortcut::activated, this, &BitcoinGUI::showRepair);
```

Comment:
15756 why are you doing this?

Reply:
as suggested by @UdjinM6 https://github.com/UdjinM6/dash/commit/422d9d1d6d9332b7be98110c54f316905a97fa92

Reply:
yes, 15756 replaces the solution we had before

Reply:
César De Jesús Castellanos CDC alministrador y propietario de proyecto services Hash tags DOCS does doesn oh Criptomonedas Bitcoin Ethereum dashboard oh Dashcore